apps/ledger-live-mobile/src/dynamicContent/utils.tsx

```diff
@@ -79,6 +79,7 @@ export const mapAsCategoryContentCard = (card: BrazeContentCard): CategoryConten
   link: card.extras.link,
   cta: card.extras.cta,
   isDismissable: Boolean(card.extras.isDismissable && card.extras.isDismissable === "true"),
+  hasPaggination: Boolean(card.extras.hasPaggination && card.extras.hasPaggination === "true"),
```

Comment:
You can use optional chaining https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
```suggestion
  hasPaggination: Boolean(card.extras?.hasPaggination === "true"),
```
If `extras` is optional you can do the same:
`card?.extras?.hasPaggination === "true"`

Reply:
Done ✅